hw/ip/otbn/data/otbn.hjson

```diff
@@ -162,9 +203,49 @@
       hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "busy",
-          desc: "OTBN is performing an operation."
+        { bits: "7:0",
+          name: "status",
+          resval: 0,
+          // Note: Keep the list of status codes in sync with status_e in
+          // otbn_pkg.sv.
+          desc: '''
+            Indicates the current state OTBN is in.
+
+            All BUSY flags represent an operation started by a write to the
+            !!CMD register.
```

Comment:
Just one small clarification:

Sometimes DMEM and IMEM secure wipe is performed as a reaction to a fatal error.
Does this mean that, in such case, we should not set signals `BUSY_SEC_WIPE_DMEM` and `BUSY_SEC_WIPE_IMEM` but only keep `BUSY_EXECUTE` until complete secure wipe is finished?

Reply:
yes. The busy flags only represent user-initiated operations (which are started by writing CMD).